monitoring/prober/conftest.py

```diff
@@ -38,6 +57,7 @@ def __init__(self, token_endpoint, service_account_json):
     credentials = service_account.Credentials.from_service_account_file(
         service_account_json).with_scopes(['email'])
     oauth_session = google_requests.AuthorizedSession(credentials)
+    oauth_session = requests.Session()
```

Comment:
Should the previous line be removed given this line?  Since this line is in ServiceAccountAuthAdapter, does authorization against a service account still work with this line added?

Reply:
This line needs to be removed, its a stray